package.json

```diff
@@ -20,11 +20,13 @@
     "@ecl/ec-component-media-container": "2.35.0",
     "@ecl/ec-preset-editor": "2.35.0",
     "@ecl/ec-preset-legacy-website": "2.35.0",
+    "@ecl/ec-preset-website": "2.35.0",
     "@ecl/ec-theme-default": "2.35.0",
     "@ecl/ec-utility-background": "2.35.0",
     "@ecl/ec-utility-border": "2.35.0",
     "@ecl/eu-component-form-help-block": "2.35.0",
     "@ecl/eu-preset-legacy-website": "2.35.0",
```

Comment:
Also from here, we can drop these two if we don't need them

Reply:
fixed